### ISSUE TITLE
No room humidity entity available 

### ISSUE THREAD

**Issue by pasvasconcelos** [rospogrigio/daikin_residential#115]
As requested by @droscy please find bellow the diagnostic data for one of my Daikin AC units:

[daikin_residential-7a28c82eb0be0e2fcfce9f9540bd8cf3-Pilar-9673d65af8800c2ae65c6f8d9430536d.json.txt](https://github.com/rospogrigio/daikin_residential/files/10669686/daikin_residential-7a28c82eb0be0e2fcfce9f9540bd8cf3-Pilar-9673d65af8800c2ae65c6f8d9430536d.json.txt)

**Comment by droscy** [comment]
Daikin cloud does not report humidity for your unit, the `sensoryData` section has only temperatures.

    "sensoryData": {
      "ref": "#sensoryData",
      "settable": false,
      "value": {
        "roomTemperature": {
          "settable": false,
          "value": 25,
          "stepValue": 1,
          "minValue": -25,
          "maxValue": 50
        },
        "outdoorTemperature": {
          "settable": false,
          "value": 11.5,
          "stepValue": 0.5,
          "minValue": -25,
          "maxValue": 50
        }
      }
    },

To be sure you could check on official Onecta app, you should't see humidity either.

**Comment by pasvasconcelos** [comment]
But with Daikin AC integration I have the humidity sensor:
![image](https://user-images.githubusercontent.com/124313154/217332049-0b7a6204-c750-4d88-976c-98b02d00b11b.png)


with Daikin Residential Controller:
![image](https://user-images.githubusercontent.com/124313154/217332444-9ca752da-58fa-4627-a5bb-e65cb7c141f7.png)

**Comment by droscy** [comment]
I believe you, but in the file you attached there is nothing releated to humidity (check yourself, open it and search `humi`, case insensitive). The Daikin cloud is not exposing the humidity sensor for your unit.

**Comment by droscy** [comment]
Your unit, as reported in the file, has the `BRP069C4x` wifi adapter, which is not compatible with Daikin AC as stated in official page https://www.home-assistant.io/integrations/daikin/, are you sure that both intergrations point to the same unit? Maybe you have one *outside* unit but two *internal* units supplied with different wifi adapters...

**Comment by pasvasconcelos** [comment]
how can I extract the diagnostic data for Daikin AC integration and for Daikin Residential Controller separately?

**Comment by pasvasconcelos** [comment]
> Your unit, as reported in the file, has the `BRP069C4x` wifi adapter, which is not compatible with Daikin AC as stated in official page https://www.home-assistant.io/integrations/daikin/, are you sure that both intergrations point to the same unit? Maybe you have one _outside_ unit but two _internal_ units supplied with different wifi adapters...

yes... the diagnostic data I sent first was for `BRP069C4x`. But the print screens was for the same unit: `BRP069B4x`.

**Comment by droscy** [comment]
> how can I extract the diagnostic data for Daikin AC integration

If there isn't any "Download Diagnositcs" button probably the integration is not offering such functionality.

> yes... the diagnostic data I sent first was for `BRP069C4x`. But the print screens was for the same unit: `BRP069B4x`.

Be careful you wrote two different wifi adapters: `BRP069*C*4x` and `BRP069*B*4x`!

The first (`C` one) is compatible with this integration **only** (and cloud data doesn't have humidity); the second (`B` model) should be compatible with both integrations and you configured with Daikin AC. Probably if you link the `B` adapter to your Daikin account you will see it in this integration too. But maybe you will have a name clash in home assistant...

However all this sounds very strange to me: if you have only one unit, why you have two wifi adapters?

**Comment by pasvasconcelos** [comment]
I have 5 indoor units (2 x 'BRP069B4x' and 3 x 'BRP069C4x'), but only your integration is compatible with all... Daikin AC only discover 2 units 'BRP069B4x', and with Daikin AC I can see humidity information for these 2 units, but with Daikin Residential Controller I can't...

![image](https://user-images.githubusercontent.com/124313154/217858589-2c18957c-290e-487e-8a56-3c906ee97188.png)

**Comment by droscy** [comment]
> but with Daikin Residential Controller I can't...

No, you can't, as long as that information is not exposed by Daikin cloud.
